terraform/gcp/modules/monitoring/infra/alerts.tf

```diff
@@ -66,6 +66,41 @@ resource "google_monitoring_alert_policy" "ssl_cert_expiry_alert" {
 
 ### Cloud SQL Alerts
 
+# Cloud SQL Database CPU Utilization > 90%
+resource "google_monitoring_alert_policy" "cloud_sql_cpu_utilization" {
+  # In the absence of data, incident will auto-close in 7 days
+  alert_strategy {
+    auto_close = "604800s"
+  }
+  combiner = "OR"
+
+  conditions {
+    condition_threshold {
+      aggregations {
+        alignment_period   = "300s"
+        per_series_aligner = "ALIGN_MEAN"
+      }
+      comparison      = "COMPARISON_GT"
+      duration        = "0s"
+      filter          = "metric.type=\"cloudsql.googleapis.com/database/cpu/utilization\" resource.type=\"cloudsql_database\""
+      threshold_value = "0.9"
```

Comment:
What do you think about two alerts, one with `severity = "warning"` at something like .80 as a warning we need to increase, and another at .90 or .95 as a page?

Reply:
I like that idea!

Reply:
I pushed that and also added low severity versions of the existing memory and disk utilization alerts.

It's a lot of duplication, but I'm guessing the only alternative is a terraform module, which doesn't seem any better.